[INFO   ] Logger: Record log in /storage/emulated/0/__Projetos__/Hive/front/app/.kivy/logs/kivy_25-02-04_0.txt
[INFO   ] Kivy: v2.2.1
[INFO   ] Kivy: Installed at "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/__init__.py"
[INFO   ] Python: v3.11.4 (main, Sep 30 2023, 10:54:38) [GCC 11.4.0]
[INFO   ] Python: Interpreter at "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/bin/python3"
[INFO   ] Logger: Purge log fired. Processing...
[INFO   ] Logger: Purge finished!
[INFO   ] KivyMD: 1.2.0, git-Unknown, 2024-12-05 (installed at "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivymd/__init__.py")
[WARNING] KivyMD: Version 1.2.0 is deprecated and is no longer supported. Use KivyMD version 2.0.0 from the master branch (pip install https://github.com/kivymd/KivyMD/archive/master.zip)
[INFO   ] Factory: 190 symbols loaded
[INFO   ] Image: Providers: img_tex, img_dds, img_sdl2, img_pil (img_ffpyplayer ignored)
[INFO   ] Text: Provider: sdl2
[INFO   ] Window: Provider: sdl2
[INFO   ] GL: Using the "OpenGL ES 2" graphics system
[INFO   ] GL: Backend used <sdl2>
[INFO   ] GL: OpenGL version <b'OpenGL ES 3.2 V@0615.91 (GIT@a63fb93269, I16b8254a37, 1724729505) (Date:08/27/24)'>
[INFO   ] GL: OpenGL vendor <b'Qualcomm'>
[INFO   ] GL: OpenGL renderer <b'Adreno (TM) 610'>
[INFO   ] GL: OpenGL parsed version: 3, 2
[INFO   ] GL: Texture max size <16384>
[INFO   ] GL: Texture max units <16>
[INFO   ] Window: auto add sdl2 input provider
[INFO   ] Window: virtual keyboard not allowed, single mode, not docked
[INFO   ] Clipboard: Provider: android
[WARNING] Factory: Ignored class "HomeScreen" re-declaration. Current -  module: None, cls: <class '__main__.HomeScreen'>, baseclass: None, filename: None. Ignored -  module: None, cls: None, baseclass: Screen, filename: None.
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "kivy/properties.pyx", line 961, in kivy.properties.ObservableDict.__getattr__
[WARNING] stderr: KeyError: 'contacts_list'
[WARNING] stderr: 
[WARNING] stderr: During handling of the above exception, another exception occurred:
[WARNING] stderr: 
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/storage/emulated/0/__Projetos__/Hive/front/app/main.py", line 91, in <module>
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/app.py", line 955, in run
[WARNING] stderr:     self._run_prepare()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/app.py", line 925, in _run_prepare
[WARNING] stderr:     root = self.build()
[WARNING] stderr:            ^^^^^^^^^^^^
[WARNING] stderr:   File "/storage/emulated/0/__Projetos__/Hive/front/app/main.py", line 87, in build
[WARNING] stderr:     HiveApp().run()
[WARNING] stderr:             ^^^^^^^^
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/uix/screenmanager.py", line 973, in __init__
[WARNING] stderr:     super(ScreenManager, self).__init__(**kwargs)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/uix/floatlayout.py", line 65, in __init__
[WARNING] stderr:     super(FloatLayout, self).__init__(**kwargs)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/uix/layout.py", line 76, in __init__
[WARNING] stderr:     super(Layout, self).__init__(**kwargs)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/uix/widget.py", line 366, in __init__
[WARNING] stderr:     self.apply_class_lang_rules(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/uix/widget.py", line 470, in apply_class_lang_rules
[WARNING] stderr:     Builder.apply(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/lang/builder.py", line 540, in apply
[WARNING] stderr:     self._apply_rule(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/lang/builder.py", line 658, in _apply_rule
[WARNING] stderr:     child = cls(__no_builder=True)
[WARNING] stderr:             ^^^^^^^^^^^^^^^^^^^^^^
[WARNING] stderr:   File "/storage/emulated/0/__Projetos__/Hive/front/app/main.py", line 63, in __init__
[WARNING] stderr:     online = ""
[WARNING] stderr:     ^^^^^^^^^^^^
[WARNING] stderr:   File "/storage/emulated/0/__Projetos__/Hive/front/app/main.py", line 73, in contact
[WARNING] stderr:     pass
[WARNING] stderr:        ^^
[WARNING] stderr:   File "kivy/properties.pyx", line 964, in kivy.properties.ObservableDict.__getattr__
[WARNING] stderr: AttributeError: 'super' object has no attribute '__getattr__'. Did you mean: '__setattr__'?
